alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -134,17 +136,17 @@ data ExUnits = ExUnits
     exUnitsSteps :: !Word64
   }
   deriving (Eq, Generic, Show) -- It is deliberate that there is NO ORD instance.
+  deriving
+    (BoundedMeasure, Measure)
+    via (InstantiatedAt Generic ExUnits)
+  deriving
+    (Monoid, Semigroup)
+    via (InstantiatedAt Measure ExUnits)
```

Comment:
These links are for context (ie I'm pretending to be the `haddock` exe here).

https://github.com/input-output-hk/cardano-base/blob/8c732560b201b5da8e3bdf175c6eda73a32d64bc/base-deriving-via/src/Data/DerivingVia.hs#L27-L47

https://github.com/input-output-hk/cardano-base/blob/8c732560b201b5da8e3bdf175c6eda73a32d64bc/measures/src/Data/Measure/Class.hs#L80

https://github.com/input-output-hk/cardano-base/blob/8c732560b201b5da8e3bdf175c6eda73a32d64bc/measures/src/Data/Measure/Class.hs#L155-L160

https://github.com/input-output-hk/cardano-base/blob/8c732560b201b5da8e3bdf175c6eda73a32d64bc/measures/src/Data/Measure/Class.hs#L187-L193